app/models/accounting/qb/updater.rb

```diff
@@ -113,6 +113,8 @@ def types
       def ar_model_for(qb_object_type)
         return Account if Account::QB_OBJECT_TYPE == qb_object_type
         return Customer if Customer::QB_OBJECT_TYPE == qb_object_type
+        return Vendor if Vendor::QB_OBJECT_TYPE == qb_object_type
+        return Department if Department::QB_OBJECT_TYPE == qb_object_type
```

Comment:
@meltheadorable in qa, I found this bug from my earlier work on importing vendors and departments. We need these lines for the objects to actually be created on import. hoping it's ok to slip the fix in here, but I can also revert the branch and open separate pr if you'd prefer that.

Reply:
This is fine